### PR TITLE
Feature/small bug fixes

### PR DESCRIPTION
**Pull request by cblignaut**
@saeedmb8: Could you please check this is changes on YourTip feedback from QA and Jess.

### CODE REVIEW

File: molo/yourtips/forms.py [hunk 1]
```diff
@@ -10,6 +10,24 @@ class YourTipsEntryForm(forms.ModelForm):
     tip_text = forms.CharField(widget=forms.Textarea(
     ))
 
+    def clean_tip_text(self):
+        data = self.cleaned_data['tip_text']
+        if len(data.split()) > 30:
+            raise forms.ValidationError(
+                "Sorry your tip is too long, "
+                "please edit it and cut down %d words." % (
+                    len(data.split()) - 30
```
Comment (smarzban) [inline molo/yourtips/forms.py]: @cblignaut we need to wrap the text in `_(....)` in order to allow the text that we will show the user to be translatable
`forms.ValidationError( 
             _("Sorry your tip is too long, please edit it and cut down %d words.") % (len(data.split()) - 30)
)`

File: molo/yourtips/forms.py [hunk 2]
```diff
@@ -10,6 +10,24 @@ class YourTipsEntryForm(forms.ModelForm):
...
+                )
+            )
+        elif len(data) > 140:
+            raise forms.ValidationError(
+                "Sorry your tip is too long, "
+                "please edit it and cut down %d characters." % (
+                    len(data) - 140
```
Comment (smarzban) [inline molo/yourtips/forms.py]: same here

File: molo/yourtips/templates/yourtips/your_tips_entry.html [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+{% get_your_tip as your_tip %}
 <div class="yourtips yourtips--page">
   <h1 class="heading heading--yourtips-page">
     {% trans 'Submit' %} {{tip_page.title|slice:":-1"}}
   </h1>
   <h3 class="yourtips__description">
-    {% if homepage_action_copy %}
-      {% trans homepage_action_copy %}
+    {% if your_tip.homepage_action_copy %}
+      {% trans your_tip.homepage_action_copy %}
```
Comment (smarzban) [inline molo/yourtips/templates/yourtips/your_tips_entry.html]: @cblignaut we don't need to wrap `your_tip.homepage_action_copy` variable in a trans block, trans block won't work on the variables as you need the text when generating a PO file. In this case admin will need to add the translated text in wagtail.

File: molo/yourtips/forms.py (hunk 1 above)
Comment (cblignaut) [reply molo/yourtips/forms.py]: Thanks, I always forget about the translatable text.

**Comment by smarzban** [comment]
👍

**Pull request merged**